src/epub/text/chapter-3.xhtml

```diff
@@ -73,7 +73,7 @@
 			<p>Diamond took her hand. It was cold, but so pleasant and full of life, it was better than warm. She led him across the garden. With one bound she was on the top of the wall. Diamond was left at the foot.</p>
 			<p>“Stop, stop!” he cried. “Please, I can’t jump like that.”</p>
 			<p>“You don’t try” said North Wind, who from the top looked down a foot taller than before.</p>
-			<p>“Give me your hand again, and I will, try” said Diamond.</p>
+			<p>“Give me your hand again, and I will try” said Diamond.</p>
```

Comment:
This comma actually must go after `try`. Can you fix that and amend his PR? You can just amend your one commit and force push, so we don't add a bunch of commits to fix one character.

Reply:
I am gonna create another pull request because I am not finding how to ammend the commit through the web and locally I am having conflit. This is alright with you?

Reply:
Sure